o/openresty/redhat_ubi8/README.md

```diff
@@ -0,0 +1,20 @@
+Build openresty/openresty
+------------------------------------
+
+Step 1)	Build openresty/openresty container image (once per release)
+	$ docker build -t openresty_ibm .
+
+Step 2) Build openresty/openresty
+
+        Usage:
+                $ docker run --rm -v `pwd`:/ws openresty_ibm bash -l -c "cd /ws; ./build.sh <rel_tag> 2>&1 | tee output.log"
+
+        Examples:
+                ==========
+                Build Only
+                ==========
+                # by default, build "lasttestedrelease" which is v1.19.3.1  marked as a 'Tagged'  as on 7 Nov 2020
+                # both the commands below build the v1.19.3.1: branch
+                $ docker run --rm -v `pwd`:/ws openresty_ibm bash -l -c "cd /ws; ./build.sh 2>&1 | tee output.log"
+                $ docker run --rm -v `pwd`:/ws openresty_ibm bash -l -c "cd /ws; ./build.sh lasttestedrelease 2>&1 | tee output.log"
+
```

Comment:
Maybe good to add a build argument to trigger tests if desired.

Reply:
Sure, will add.